index.html

```diff
@@ -59,60 +90,210 @@ <h2 id="overview">Overview</h2>
       cohesive resource.
     </p>
 
+    <h3>Who are we?</h3>
+
     <p>
-      We are the Integration and Coordination Center (ICC), a center within the
-      CFDE responsible for, among other things, collecting metrics associated
-      with Common Fund projects and providing them to the NIH. We do this in
-      close collaboration with other Centers and NIH program staff.
+      <a href="#contact">The authors of this page</a> are the Integration and
+      Coordination Center (ICC), a center within the CFDE.
     </p>
 
     <p>
-      This page contains instructions on how to
-      <strong>submit your Common Fund project</strong>
-      so that we can properly collect the required metrics on it.
+      We are responsible for (among other things) gathering info associated with
+      Common Fund projects and providing it to the NIH. We also aim to make
+      already-public info more accessible by presenting it in a centralized way.
     </p>
 
-    <h2 id="contact">Contact</h2>
+    <p>
+      We do not operate alone; we do this in close collaboration with other
+      centers and the NIH.
+    </p>
+
+    <h3>What is this page?</h3>
 
     <p>
-      If you have any questions, suggestions, comments, or feedback, please file
-      a
-      <a href="../../issues/new">new issue</a>, or contact one of the current
-      maintainers/team members directly:
+      This page explains how you can share some
+      <strong><em>extra info</em> about your Common Fund project</strong>,
+      helping the NIH have a more complete understanding of your work.
     </p>
 
-    <ul>
-      <li>
-        <a href="mailto:casey.s.greene@cuanschutz.edu">Casey Greene</a> -
-        Evaluation Core Lead
-      </li>
-      <li>
-        <a href="mailto:sean.2.davis@cuanschutz.edu">Sean Davis</a> - Evaluation
-        Core Lead
-      </li>
-      <li>
-        <a href="mailto:vincent.rubinetti@cuanschutz.edu">Vincent Rubinetti</a>
-        (<a href="https://github.com/vincerubinetti">@vincerubinetti</a>) -
-        Software developer
-      </li>
-    </ul>
+    <p>
+      <a href="#how-does-it-work"
+        >See below for exactly what this extra info is</a
+      >.
+    </p>
 
-    <h2 id="submit-your-project">Submit your project</h2>
+    <h3>What do we do with this info?</h3>
 
     <p>
-      Many metrics about Common Fund projects &ndash; like grant details,
-      publications, and citations &ndash; we can already automatically collect
-      from public NIH resources. However, other metrics
-      <strong>require some manual steps from project teams</strong>
-      before we can collect them.
+      We do not judge or evaluate your projects in any way, we only gather info
```

Comment:
```suggestion
      We gather info about projects, summarize it,
```

Reply:
I've put in the "summarize" but can you elaborate on removing the "judge or evaluate"? I thought it was good to explicitly say that we're not looking at it very hard.

Reply:
Actually, I'm thinking we shouldn't use "summarize" because someone could be worried that we're not "summarizing it in the right way". I think maybe I originally left this vague on purpose to indicate we're just unopinionated-ly gathering stuff and giving it to the NIH to make of it what they will.

Reply:
I like that sentiment - I was avoiding "judge or evaluate" to keep those words out - we also need to strike a balance because we are the evaluation core, so to an extent there is evaluation. For example, if someone hosts a training and gets a spike in access/use, we'd "evaluate" that the training had a positive influence.

Reply:
Okay in the interest of time, I think I'll leave it vague... my thinking: better to leave it vague to leave room for clarification than to say something possibly misleading.

Reply:
works for me